static/css/base.scss

```diff
@@ -433,3 +433,9 @@ button.ui.button.streamelements {
 img.emote {
   height: 2em;
 }
+
+.logo {
+  max-width: 64px;
+  max-height: 64px;
+  border-radius: 0.28571429rem;
```

Comment:
magic number? what is this number based on

Reply:
The radius of the search box